addon/bootstrap.js

```diff
@@ -119,22 +119,26 @@ function shutdown(addonData, reason) {
   // FRAGILE: handle uninstalls initiated by USER or by addon
   if (reason === REASONS.ADDON_UNINSTALL || reason === REASONS.ADDON_DISABLE) {
     log.debug("uninstall or disable");
+
```

Comment:
Can you add a note here to say we check if `feature` exists because it's possible the study is shutting down before it has instantiated the feature. Ex: if the user is ineligible or if the study has expired.

Reply:
Clarifying this